docker-compose.test.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:latest
+    env_file:
+      - .env
+    volumes:
+      - postgres_data_test:/var/lib/postgresql/data
+
+  test:
+    build:
+      context: .
+      dockerfile: Dockerfile.test
+    image: anem-per-feina:tests
+    environment:
+      SECRET_KEY: secret
```

Comment:
This should not be hardcoded, and should load the same definition than the service `db`